bin/copy_peste

```diff
@@ -3,8 +3,8 @@
 # Simple CopyPeste launcher that runs it with or without parameters.
 # Right now, the only allowed parameter is -d or --debug.
 
```

Comment:
"... new arguments" *

Yup, will do that. I didn't know there was any referenced there.